### PR TITLE
OLS-1198: Remove drop down used to ask a question

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Affects:
[lightspeed-main](https://github.com/openshift/openshift-docs/tree/lightspeed-docs-main)
[lightspeed-docs-1.0](https://github.com/openshift/openshift-docs/tree/lightspeed-docs-1.0)

This PR is part of the standalone doc set for the Lightspeed project. The upcoming January release will see the product go from TP to GA. Kathryn is aware that content previously applied for a product that is part of a Tech Preview release. The project is seeking feedback from early adopters. Now, we are preparing for GA.

PR must be CP'd back to the 1.0 branch.

Version(s): 1.0 

Issue: https://issues.redhat.com/browse/OLS-1198

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by kyoto** [comment]
LGTM

**Comment by JoaoFula** [comment]
LGTM

**Comment by rh-tokeefe** [comment]
ATTENTION MERGE REVIEWER: This PR simply deletes two topics that are no longer needed. The PR had received dev and QA review. My understanding is that peer review is not required. 

If you do not think this is correct for this PR, please tag me in a comment and I will submit the PR for peer review. Thanks!

File: operate/ols-using-openshift-lightspeed.adoc [hunk 1]
```diff
@@ -15,8 +15,6 @@ There are different ways you can access the {ols-official} user interface.
 The following topics provide information about submitting questions to {ols-full}.
 
 include::modules/ols-initiating-chat-using-chat-window.adoc[leveloffset=+1]
-include::modules/ols-initiating-chat-using-actions-list.adoc[leveloffset=+1]
-include::modules/ols-initiating-chat-using-node-options-icon.adoc[leveloffset=+1]
```
Comment (adellape) [inline operate/ols-using-openshift-lightspeed.adoc:19]: If these files are no longer in use by anything (which seems to be the case on this branch), then they can be fully removed/deleted from the branch. If you think they might be useful in the future, then maybe could move them to the `unused_topics/` folder for now.

**Review by adellape — COMMENTED** [review]
> This PR simply deletes two topics that are no longer needed. The PR had received dev and QA review. My understanding is that peer review is not required.

I'm not sure if that specific scenario has been articulated either way anywhere, but I would still assume peer review -> merge review. Happy to be wrong tho.

In any case, see comment about removing the actual files. I'll count this as `peer-review-done`, but please re-queue for merge-review when updated. Thanks!

File: operate/ols-using-openshift-lightspeed.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply operate/ols-using-openshift-lightspeed.adoc:19]: Thanks for this suggestion...as well as the notion that peer review should still be done.

Comment (rh-tokeefe) [reply operate/ols-using-openshift-lightspeed.adoc:19]: Pushed change to PR. 

Done.

**Review by adellape — APPROVED** [review]

**Pull request merged**